thentos-core/thentos-core.cabal

```diff
@@ -43,7 +43,7 @@ library
   hs-source-dirs:
       src
   ghc-options:
-      -Wall
+      -Wall -j5
```

Comment:
Ok, but isn't 5 a more sensible default than 1?  (I'm assuming you can override this with `configure`?)